CONTRIBUTING.md

```diff
@@ -1,126 +1,192 @@
-# How to contribute
-
-k8gb is Apache 2 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into k8gb project.
-
-# Slack chat
+<!-- omit in toc -->
+# Contributing to k8gb
+
+- [Getting started](#getting-started)
+- [Getting help](#getting-help)
+- [Reporting issues](#reporting-issues)
+- [Contribution flow](#contribution-flow)
+- [Local setup](#local-setup)
+  - [Deploy k8gb locally](#deploy-k8gb-locally)
+  - [Upgrade k8gb to local candidate](#upgrade-k8gb-to-local-candidate)
+- [Testing](#testing)
+- [Debugging](#debugging)
+- [Code style](#code-style)
+- [Commit and Pull Request message](#commit-and-pull-request-message)
+  - [Signature](#signature)
+  - [Changelog](#changelog)
+- [Documentation](#documentation)
+- [k8gb.io website](#k8gbio-website)
+  - [Local website authoring and testing](#local-website-authoring-and-testing)
 
-- Find us at `#sig-multicluster` on kubernetes.slack.com
+k8gb is licensed under [Apache 2 License](./LICENSE) and accepts contributions via GitHub pull requests.
+This document outlines the resources and guidelines necessary to follow by contributors to the k8gb project.
 
 ## Getting started
 
 - Fork the repository on GitHub
 - See the [local playground guide](/docs/local.md) for testing environment setup
 
-## Reporting bugs and creating issues
+## Getting help
+
+Find us at `#sig-multicluster` on kubernetes.slack.com
```

Comment:
While we are on it, can we please also mention https://github.com/k8gb-io/k8gb/discussions here?

Reply:
I also wonder if we can get a dedicated `#k8gb` channel at CNCF Slack (https://cloud-native.slack.com/) as CNCF project. See https://github.com/argoproj/argoproj/issues/47

Reply:
@ytsarev amended with the project GH discussions.

Reply:
@somaritane great idea, I've just created `#k8gb` channel there, please amend the docs

Reply:
@ytsarev Done